Doc/library/trace.rst

```diff
@@ -96,6 +96,15 @@ Modifiers
    Directory where the report files go.  The coverage report for
    ``package.module`` is written to file :file:`{dir}/{package}/{module}.cover`.
 
+.. cmdoption:: -d, --trace_fd
```

Comment:
Mixing `-` and `_` looks weird. Either use minus, `--trace-fd` (like `--no-report`) or omit the separator, `--tracefd` (like `--listfuncs`, `--trackcalls`, `--coverdir`).

Also specify that the option takes an argument: `--tracefd=<fd>`.

I am not sure that it is necessary to add a short option. The set of letters is limited, and `-d` could be used for more popular option ("directory" or "description" or "delete").